src/App.tsx

```diff
@@ -88,7 +89,7 @@ function App() {
             }
             exact
           />
-
+          <GsuiteProtectedRoute path="/gsuite" Component={Welcome} exact />
```

Comment:
Probably use the `Authorize.tsx` component instead of the welcome component. It has the necessary logic to show a loading screen, refresh page if not authenticated & save tokens to recoil if authenticated. You could also create a new component to display on that route.

Reply:
Need to create new component for admin portal, but using Welcome as stand in for now.

Reply:
`Authorize.tsx` is being used in the case where user `is_verified: false` in `GsuiteProtectedRoute`